Appraisals

```diff
@@ -1368,7 +1368,8 @@ elsif ruby_version?('3.0') || ruby_version?('3.1')
     gem 'ethon'
     gem 'excon'
     gem 'grape'
-    gem 'graphql', '>= 2.0'
+    # gem 'graphql', '>= 2.0'
+    gem 'graphql', git: 'https://github.com/TonyCTHsu/graphql-ruby.git', branch: 'datadog/unified-tagging'
```

Comment:
Can you make the upstream PR non-draft? So we can unblock this versioning change when graphql releases a new version?

Reply:
@TonyCTHsu was this done? If so, could we link the PR here in this GH PR (probably in the PR description at the top).